### ISSUE TITLE
Looking for GAN loss parameters

### ISSUE THREAD

**Issue by fabio-C** [NVIDIA/pix2pixHD#41]
In pix2pix [tensorflow implementation](https://github.com/affinelayer/pix2pix-tensorflow) I can set the following parameters:
--**l1_weight**: weight on L1 term for generator gradient
--**gan_weight**: weight on GAN term for generator gradient"

They are used to create the GAN loss, in this way:
`gen_loss = gen_loss_GAN * a.gan_weight + gen_loss_L1 * a.l1_weight`
Row 455 of [pix2pix.py](https://github.com/affinelayer/pix2pix-tensorflow/blob/master/pix2pix.py)

Now I am moving to pix2pixHD, but I can't find these parameters in this implementation. How can I modify it? I didn't find also the same equation in this code.

**Comment by tcwang0509** [comment]
We don't have L1 loss used, so there's no weight associated with it. We do have an L1 feature matching loss, whose weights can be changed by '--lambda_feat'.
